src/cli.ts

```diff
@@ -64,9 +64,7 @@ const hasValidToken = (): boolean => {
   if (!token) { return false }
 
   const decoded = decode(token)
-  if (!decoded || !decoded.exp || Number(decoded.exp) < (Date.now() / 1000)) { return false }
-
-  return true
+  return decoded && typeof decoded !== 'string' && decoded.exp && Number(decoded.exp) >= Date.now() / 1000
```

Comment:
This is kinda strange, I don't get what `decoded.exp` is supposed to be, but please double check this expression's correctness

Reply:
I just inverted the condition above to return the result directly. It should be correct, or toolbelt would keep asking me to log in again, even when my token is valid.

Also, for the record: `decoded.exp` is a unix timestamp. It is directly comparable to `Date.now() / 1000`, which is a number as well.
![JS console showing that both values are comparable](https://user-images.githubusercontent.com/6957289/58436417-a6e7b500-809b-11e9-91ea-fc804c5376d7.png)